target/linux/realtek/config-5.10

```diff
@@ -85,7 +85,9 @@ CONFIG_HZ_PERIODIC=y
 CONFIG_I2C=y
 CONFIG_I2C_ALGOBIT=y
 CONFIG_I2C_BOARDINFO=y
+CONFIG_I2C_CHARDEV=y
 CONFIG_I2C_GPIO=y
+CONFIG_I2C_RTL9300=y
```

Comment:
This enables both the driver in this commit, as well as userspace access to I2C devices. In my opinion, at least the latter should be a separate commit with argumentation why it needs to be enabled.

Reply:
I'll split this up. The /dev/i2c- devices are extremely useful, as they allow to use the i2c tools, e.g. to identify the SFP modules, fans and thermal probes as well as other tools accessing these devices. Up to until now we did not have a i2c device available on the platform, only sfp modules accessed via gpio, which did not expose an i2c device usable outside of the sfp module driver.